packages/node/src/schematics/application/application.ts

```diff
@@ -144,19 +144,35 @@ function addProxy(options: NormalizedSchema): Rule {
     const projectConfig = getProjectConfig(host, options.frontendProject);
     if (projectConfig.architect && projectConfig.architect.serve) {
       const pathToProxyFile = `${projectConfig.root}/proxy.conf.json`;
-      host.create(
-        pathToProxyFile,
-        JSON.stringify(
-          {
-            '/api': {
-              target: 'http://localhost:3333',
-              secure: false,
+
+      if (!host.exists(pathToProxyFile)) {
+        host.create(
+          pathToProxyFile,
+          JSON.stringify(
+            {
+              [`${options.name}`]: {
+                target: 'http://localhost:3333',
+                secure: false,
+              },
             },
+            null,
+            2
+          )
+        );
+      } else {
+        //add new entry to existing config
+        const proxyFileContent = host.get(pathToProxyFile).content.toString();
+
+        const proxyModified = {
+          ...JSON.parse(proxyFileContent),
+          [`${options.name}`]: {
```

Comment:
For the next applications I think it will be clearer if we keep using a path format like `/api-${options.name}` or somethinh similar, instead of just the name. WDYT?

Reply:
Thats a good idea. I appended -api to the name as the key